src/utils/ceramicBootstrap.js

```diff
@@ -57,7 +58,7 @@ async function run() {
   const seed = new Uint8Array(randomBytes(32))
   console.log("Created seed", seed)
   // Connect to the local Ceramic node
-  const ceramic = new Ceramic(CERAMIC_URL_TESTNET)
+  const ceramic = new Ceramic(CERAMIC_URL)
```

Comment:
We need to add the Ceramic UR in `network.json` and use based in the selected network.
test: `https://ceramic-clay.safient.io`
mainnet: test: `https://ceramic.safient.io`
local and dev: test: `http://0.0.0.0:7007`